site_config_client/__init__.py

```diff
@@ -25,7 +26,9 @@ def list_sites(self):
         """
         Returns a list of all Sites
         """
-        response = requests.get(self.build_url('v1/site/'))
+        auth_headers = {'Authorization': '{}'.format(self.api_token)}
```

Comment:
It has to be `Token` or `Bearer`, right?

```suggestion
        auth_headers = {'Authorization': 'Token {}'.format(self.api_token)}
```

Reply:
@OmarIthawi thank you and good call! it works with `Token`